### DISCUSSION TITLE
when will we get GNOME 40 support?

### DISCUSSION THREAD

**Discussion by cooldoglol** [dnschneid/crouton#4517] [Q&A]
modern GNOME 3 versions for some reason are not working. on kali i get a crash and i cant use GNOME.
so, yeah i got 2 questions with that
1:when will GNOME 3 work or how do i get it working on my chromebook?
2:when will we get the ability to use GNOME 40 instead off GNOME 3?
sorry for the short discussion, but i just want an anser to when will we get GNOME 40 and how do i get GNOME 3 working?
if i can i might file a pull request for GNOME 40

**Comment by ghost** [comment]
In theory you could get GNOME 40 to work. Would it take a lot of work and time? Yes.
